at_response.c

```diff
@@ -1229,6 +1229,8 @@ static int at_response_cmgr (struct pvt* pvt, const char * str, size_t len)
 				return 0;
 			}
 
+			ast_log(LOG_NOTICE, "[%s] Incoming message parsed '%s'\n", PVT_ID(pvt), str);
+
 			ast_debug (1, "[%s] Successfully read SMS message\n", PVT_ID(pvt));
```

Comment:
How about ast_verbose then?

You can (and I always do) set verbose mode to a high value at startup through asterisk.conf.